docs/sources/developers/plugins/data-frames.md

```diff
@@ -6,15 +6,13 @@ title: Data frames
 
 Grafana supports a variety of different data sources, each with its own data model. To make this possible, Grafana consolidates the query results from each of these data sources into one unified data structure called a _data frame_.
 
-The data frame structure is a concept that's borrowed from data analysis tools like the [R programming language](https://www.r-project.org), and [Pandas](https://pandas.pydata.org/).
-
-> Data frames are available in Grafana 7.0+, and replaced the Time series and Table structures with a more generic data structure that can support a wider range of data types.
-
-This document gives an overview of the data frame structure, and of how data is handled within Grafana.
+> **Note:** Data frames are available in Grafana 7.0+, and replaced the Time series and Table structures with a more generic data structure that supports a wider range of data types.
```

Comment:
At what stage can we just remove notes like this... We have versioned docs, and this is now a 3 year old notice?

Reply:
I think it's safe to remove, but I also think you asked a great question about versioning notes relevancy and I'm looking into this.

Reply:
Removed.